x/debug/options.go

```diff
@@ -0,0 +1,44 @@
+package debug
+
+import "go.uber.org/zap"
+
+// Option describes a func that can modify options.
+type Option interface {
+	apply(*options)
+}
+
+type optionFunc func(*options)
+
+// opts represents the combined options supplied by the user.
+type options struct {
+	logger   *zap.Logger
+	template templateIface
+}
+
+// Logger specifies the logger that should be used to log.
```

Comment:
This is no longer commented as to the default value, and the default value is no longer a public variable per what @akshayjshah and I discussed for self-documentation